packages/config/vite/package.json

```diff
@@ -14,15 +14,16 @@
     }
   },
   "scripts": {
-    "build": "NODE_OPTIONS='--max-old-space-size=16384' tsup",
-    "dev": "NODE_OPTIONS='--max-old-space-size=16384' tsup --watch",
+    "build": "cross-env NODE_OPTIONS='--max-old-space-size=16384' tsup",
+    "dev": "cross-env NODE_OPTIONS='--max-old-space-size=16384' tsup --watch",
     "lint": "eslint . --cache --max-warnings 0",
     "typecheck": "tsc --noEmit --tsBuildInfoFile .tsbuildinfo"
   },
   "dependencies": {
     "@vitejs/plugin-react": "^4.2.1",
     "vite": "^5.2.8",
-    "vite-tsconfig-paths": "^4.3.2"
+    "vite-tsconfig-paths": "^4.3.2",
+    "cross-env": "^7.0.3"
```

Comment:
This PR is great!, completely went over my head while doing that. Just one little change, because cross-env would then be something that is used widely among the monorepo, it would be better to have it listed in the `devDependencies` of the root `package.json` file instead of having it in basically every package where it is used, this is for example how `eslint` as a dependency is used among the monorepo.